### ISSUE TITLE
XML encoder isnt working correctly with special characters

### ISSUE THREAD

**Issue by Iaggoq** [TotalCross/totalcross#190]
## Issue Title
XML encoder isnt working correctly with special characters

## Describe the bug
Totalcross's xml reader isnt reading/breaking with special characters like " ´ ", " ~ ".  

## To Reproduce
```
		public void tagName(int a,String arg0,AttributeList atts) {
			AttributeList.Iterator it = atts.new Iterator();
			while (it.next()) {
				auxNodeSax.inserts(it.getAttributeName(), it.getAttributeValue());
			}
		}

```
it.getAttributeValue() isnt returning the correct enconding

## Expected behavior
Its expected it to return some of special characters

## Devices:
Please complete the following information:
 - Device: Simulador, Rasp
 - OS: Windows
 - OS Version: 10 2004
 - TotalCross Version: 6.1.1

**Comment by Iaggoq** [comment]
Update on issue the problem its in the parser which isnt parsing corretly utf-16 characters only work corretly with utf-8, because of this some characters isnt supported

**Comment by ricardobna** [comment]
Closing this issue as the fix is on #227
